content/en/contributing/multi-account-region-testing.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Multi account and region testing"
+weight: 7
```

Comment:
I discussed this with @HarshCasper since I wasn't sure about this. @alexrashed do you think it should be 8 here?

Reply:
I don't have a strong opinion here, but I would say the parity testing guide is a bit more important (because basically _every_ PR needs to use it, while this guide is only interesting for code touching the inter-service-communication).
The parity guide has weight 7, the terraform testing has 10, so I think 8 would work well.
But again, no strong opinion :)